packages/contracts-bedrock/test/L1/OPContractsManager.t.sol

```diff
@@ -148,3 +155,29 @@ contract OPContractsManager_InternalMethods_Test is Test {
         vm.assertEq(expected, actual);
     }
 }
+
+contract OPContractsManager_Upgrade_Test is CommonTest {
+    function setUp() public override {
+        super.setUp();
+        if (!isForkTest()) {
+            // This test is only supported in forked tests, as we are testing the upgrade.
+            vm.skip(true);
+        }
+    }
+
```

Comment:
add sad path tests: 
- invalid inputs
- mismatched SuperchainConfig
- auth failures (not the `ProxyAdmin.admin()`)